qa/rpc-tests/thinblocks.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2015 The Bitcoin Core developers
+# Copyright (c) 2015-2016 The Bitcoin Unlimited developers
```

Comment:
I've just applied @awemany changes to release, so I guess the same change need to be done for `dev`. Another point since this is a BU only test I wonder if we could remoe Core copyright entirly.